packages/theme/src/fonts/index.ts

```diff
@@ -40,29 +68,41 @@ export const primary = {
 
 export const secondary = {
   family: 'Mulish, sans-serif',
-  display1: {
-    fontSize: '7.4rem',
-  },
-  display2: {
-    fontSize: '6.6rem',
-  },
-  h1: {
-    fontSize: '5.2rem',
-  },
-  h2: {
-    fontSize: '4.6rem',
-  },
-  h3: {
-    fontSize: '3.6rem',
-  },
-  h4: {
-    fontSize: '2.9rem',
-  },
-  h5: {
-    fontSize: '2.6rem',
-  },
-  h6: {
-    fontSize: '2rem',
+  display: {
+    large: {
+      fontSize: '7.4rem',
+      lineHeight: '9.2rem',
+    },
+    medium: {
+      fontSize: '6.6rem',
+      lineHeight: '8.2rem',
+    },
+  },
+  heading: {
+    large: {
+      fontSize: '5.2rem',
+      lineHeight: '6.5rem',
+    },
+    medium: {
+      fontSize: '4.6rem',
+      lineHeight: '5.8rem',
+    },
+    small: {
+      fontSize: '3.6rem',
+      lineHeight: '4.5rem',
+    },
+    xsmall: {
+      fontSize: '2.9rem',
+      lineHeight: '3.6rem',
+    },
+    xxsmall: {
+      fontSize: '2.6rem',
+      lineHeight: '3.2rem',
+    },
+    xxxsmall: {
```

Comment:
`xxxsmall` feels less semantic than `h6`

Reply:
Is it worth it to mix syntaxes i.e. `xxsmall -> h5` & `xxxsmall -> h6`?

Reply:
If they are for h1 - h6, then this promotes understandability (readability + mental model)

Reply:
Updated to h1 - h6 labels